plugins/modules/docker_image.py

```diff
@@ -463,6 +480,9 @@ def absent(self):
             if not self.check_mode:
                 try:
                     self.client.remove_image(name, force=self.force_absent)
+                except NotFound as dummy:
```

Comment:
Since we are not really using it, I'd remove the `as dummy` part altogether. It does not make the code better at all.

Reply:
Fixed in b3b6da6165168176d99abfc0d7d0e8ee3a18cfb5.